test/fix_sample/main.go

```diff
@@ -0,0 +1,6 @@
+package main
+
+func main() { 
+	println(  
+"hello") 
```

Comment:
It's not totally unexpected that committing intentionally broken Go code into the repo will trigger the linter :sweat_smile: 

I couldn't find any tests for fixing. Do they really not exist or have I just missed them?

I had started writing a unit test, but that wouldn't have been able to reproduce the failure because the failure only occurs in combination with other processors.

I can remove the extra commit with this file.

Reply:
the tests are here https://github.com/golangci/golangci-lint/tree/master/test/testdata/fix

Reply:
Thanks. I've ripped out the manual test case and modified the testing to run one test case (arbitrarily picking the first one, based on alphabetical sorting) with `--path-prefix`. I checked that this breaks with master and works in this PR.